### ISSUE TITLE
.org redesign: homepage [design] overview issue

### ISSUE THREAD

**Issue by amberleyromo** [gatsbyjs/gatsby#9526]
_(This issue is **design-focused**, not implementation-focused.)_

The below represent the components of the homepage that require design attention for the .org redesign effort:

- [x] Ecosystem #9709
- [x] Blog section #9711
- [ ] “Used by” #9752
- [ ] Showcase #9754

## Blocked / Requires discussion
- [ ] Hero content / CTA buttons #9748
- [ ] Features / "Why" #9755
- [ ] Diagram #9525

## Shelved
- [ ] Community / Creators #9753 

We will create separate issues for these subtasks as they are tackled.

@fk these capture the way you described separating out this work. Does this still look accurate?

**Comment by fk** [comment]
Will break down components in a minute and add a couple.

Also just stumbled upon https://github.com/ichabod-cranes/mansion/issues/987, should we close this one?

**Comment by amberleyromo** [comment]
Closing ichabod-cranes/mansion#987 -- thanks!

**Comment by fk** [comment]
Argh, so much for "in a minute". Sorries! :(

**Comment by fk** [comment]
@amberleyromo @greglobinski Finally got to create bespoke issues for all work, and updated the issue description above.

While I'm treating those sections as separate components that do not require a certain order, it would be nice to define the content order. Apart from "hero content", IMO the positions/priority of "Features" and "Diagram" are pretty much set in stone—I'm wondering what your thoughts are in regards to the order of

- Ecosystem
-  Blog section
-  Community / Creators
-  Showcase

**Comment by amberleyromo** [comment]
I think this is more of a content / IA question for @shannonbux and @lindaleebumblebee. First blush I agree with the way you've ordered it in your comment.

**Comment by shannonbux** [comment]
I will have all my UX research interviews on the homepage done by next Friday (Nov 16th) so will have a more informed opinion by then! As with every project, this is a balance of meeting two goals:
- internal goals (what path do we *want* people to go down)?
- user goals (what path do users want to go down and how can we help them get there?)

My first impression is that this is a lot for a homepage to fit and I'd recommend a simpler setup instead of trying to fit it all. I don't know who the ultimate decision maker is here so take this for what it's worth :)

**Comment by amberleyromo** [comment]
If that's the case, then I think community/creators should drop off. We have a lot more work to do in that domain defining it before we can (in an informed way) think about how it might relate to the homepage.

Also, @KyleAMathews -- I think it was you who first suggested pulling featured Gatsby sites into the homepage. Do you have an opinion about where it goes? Now that I'm remembering that, I'm thinking it should go more to the top.

**Comment by fk** [comment]
> My first impression is that this is a lot for a homepage to fit and I'd recommend a simpler setup instead of trying to fit it all. 

@shannonbux I can follow your concern.
FWIW [here's the internal issue comment](https://github.com/ichabod-cranes/mansion/issues/987#issuecomment-415492200) in which I initially captured the content of the homepage and I'm still basing the sections listed above on.
I did abandon this "more compact" version of the layout (trying to fit multiple subsections into row columns) because I felt it was too "dense."

**Comment by shannonbux** [comment]
@fk yes, I can see why you'd feel it was too dense. I agree! I think having
a long homepage is fine. Probably the most important question is actually
just what goes above the fold, since that's what most people focus on, at
least the first visit to the site :) And to have a more refined opinion on
that, I'll need to do 3 more interviews :)

**Comment by lindaleebumblebee** [comment]
I'm starting to lose track of all these issues about the .org homepage (a github newbie), but I will say that the 'above the fold' is the most important section of the homepage and we should keep it simple and have 1 CTA (with a possible 2nd CTA). Since our audience is developers, the 'Get Started' CTA still makes sense as we want people to build Gatsby sites!

**Comment by fk** [comment]
I just realized that we might want to tease the store on the new homepage!? 😓

**Comment by lindaleebumblebee** [comment]
It's not a bad idea to let visitors of .com know that we have a swag store, but if we go that route, I'd like to keep it as very much a lower priority CTA (at the bottom of the page or elsewhere) as we need to keep the main CTA(s) above the fold. Thanks for thinking of it Flo!

**Comment by marisamorby** [comment]
This is blocked on research and messaging, so I will pass this over once we are on to next steps.
